tests/e2e/state.go

```diff
@@ -694,6 +698,23 @@ func (tr TestRun) getRegisteredConsumerRewardDenoms(chain chainID) []string {
 	return rewardDenoms
 }
 
+func (tr TestRun) getPendingPacketQueueSize(chain chainID) uint {
+	//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
+	cmd := exec.Command("docker", "exec", tr.containerConfig.instanceName, tr.chainConfigs[chain].binaryName,
+
+		"query", "ccvconsumer", "throttle-state",
+		`--node`, tr.getQueryNode(chain),
+		`-o`, `json`,
+	)
+	bz, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatal(err, "\n", string(bz))
+	}
+
+	packetData := gjson.Get(string(bz), "packet_data_queue").Array()
```

Comment:
Could you add validation logic here? I'd be worried this could break in "interesting" ways.

From the gjson documentation on the Get function https://godocs.io/github.com/tidwall/gjson#Get:
"This function expects that the json is well-formed, and does not validate. Invalid json will not panic, but it may return back unexpected results. If you are consuming JSON from an unpredictable source then you may want to use the Valid function first. "

Reply:
Yup seems reasonable to panic when `valid` returns false for the json 👍 https://github.com/cosmos/interchain-security/commit/6e4768ee75d67fab0c6b13dd4fd17fe4720c9a8a